packages/core/components/Actor.jsonld

```diff
@@ -13,6 +13,11 @@
           "range": "cc:Bus",
           "unique": true,
           "required": true
+        },
+        {
+          "@id": "cc:Actor/beforeActor",
```

Comment:
why wasn't this called "dependency"?

Reply:
To make the direction of dependency a bit clearer. For example, `dependency` may be interpreted as `afterActor` as well.